java/src/hu/holyoil/crewmate/IStepping.java

```diff
@@ -0,0 +1,10 @@
+package hu.holyoil.crewmate;
+
+public interface IStepping {
```

Comment:
I found no better name for this, if anyone has any ideas, hmu.

Reply:
Hmmm, still reviewing the code, but maybe we don't need a new interface. Because after all, all the IStepping entities could be ISteppable too.
The settler steps just like a Controller steps. (but settler needs outside interaction)